voting/src/general_meeting.rs

```diff
@@ -225,6 +226,17 @@ impl GeneralMeetingManager for Context {
         };
         Ok(vote_box)
     }
+
+    fn set_vote_box(&mut self, meeting_id: &GeneralMeetingId, vote_box: &VoteBox) -> Result<(), Error> {
+        let key = crate::generate_voting_box_key(meeting_id);
+
+        if !self.storage().has(key.as_slice()) {
+            return Err(Error::VoteBoxNotFound)
+        }
+
+        self.storage_mut().set(key.as_slice(), serde_cbor::to_vec(&vote_box).unwrap());
+        Ok(())
+    }
```

Comment:
Returning error if votebox is not exist when setting seems strange.
How about `update_vote_box`?

When I read the `set`, I feel like it will success in the both case there is the previous state or empty.